homeassistant/components/google/calendar.py

```diff
@@ -54,12 +55,13 @@ class GoogleCalendarData:
     """Class to utilize calendar service object to get next event."""
 
     def __init__(self, calendar_service, calendar_id, search,
-                 ignore_availability):
+                 ignore_availability, max_results):
         """Set up how we are going to search the google calendar."""
         self.calendar_service = calendar_service
         self.calendar_id = calendar_id
         self.search = search
         self.ignore_availability = ignore_availability
+        self.max_results = max_results
```

Comment:
This is going to be none since there's no default set, will the library accept that?

Reply:
There is a default. It's initialized in 
`
DEFAULT_GOOGLE_SEARCH_PARAMS = {
    'orderBy': 'startTime',
    'maxResults': 5,
    'singleEvents': True,
}
`
and used in:
`
        if self.max_results:
            params['max_results'] = self.max_results
`